src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -845,6 +845,7 @@ export class BaseEditCourseForm extends React.Component {
             collapsiblesOpen={this.state.collapsiblesOpen}
             onToggle={(index, value) => this.toggleCourseRun(index, value)}
             {...this.props}
+            validate={() => {}} // override method from our props, we don't want to pass it down
```

Comment:
This does not prevent validation from occuring inside runs. I tested both course fields and run fields (manually checked and html5-validated for each).

Reply:
To fully explain:

So redux was coming up with errors from the form, but they weren't real, it was like an errors object like { course_runs: { _error: {} } } or something
But that tricks redux, despite not being a real error
It expects just {} on all-good
And it seemed to be because we were passing down the top level validate prop into the FieldArray for the runs
So I stopped passing it down by overriding with a dummy validate
And yet, somehow the fields in the FieldArray are still validated...
I don't fully understand it, tbh